book/listings/gobject_properties/4/custom_button/imp.rs

```diff
@@ -80,7 +80,7 @@ impl WidgetImpl for CustomButton {}
 // Trait shared by all buttons
 impl ButtonImpl for CustomButton {
     fn clicked(&self, button: &Self::Type) {
-        let incremented_number = self.number.borrow().clone() + 1;
+        let incremented_number = *self.number.borrow() + 1;
```

Comment:
Interesting, we should run clippy for the book listings as well
cc @Hofer-Julian

Reply:
Good idea!
Wann open a PR or issue for that?